gilt/config.py

```diff
@@ -26,6 +26,7 @@
 
 import giturlparse
 import yaml
+import interpolation
```

Comment:
Can you move the import down a line and import as `from gilt import interpolation`.  It's just a pattern we follow throughout.